src/chains/ethereum/ethereum/tests/helpers/getProvider.ts

```diff
@@ -13,16 +13,16 @@ const getProvider = async (
   }
 ) => {
   options.chain = options.chain || {};
+  options.miner = options.miner || {};
   options.logging = options.logging || { logger: { log: () => {} } };
 
   // set `asyncRequestProcessing` to `true` by default
   let doAsync = options.chain.asyncRequestProcessing;
   doAsync = options.chain.asyncRequestProcessing =
     doAsync != null ? doAsync : true;
 
-  // don't write to stdout in tests
-  if (!options.logging.logger) {
```

Comment:
Why are we removing this?

Reply:
It didn't appear to actually do anything in our tests (I replaced it with `throw new Error...` and all tests still passed)